Sources/Views/Cells/MediaMessageCell.swift

```diff
@@ -64,5 +69,7 @@ open class MediaMessageCell: MessageCollectionViewCell {
         default:
             break
         }
+        
+        displayDelegate.configureMediaMessageImageView(imageView, for: message, at: indexPath, in: messagesCollectionView)
```

Comment:
Interesting approach. Let me think about this some more. My concern is this then overwriting the `image` in the `.photo` enum. 🤔